src/CompanyService.Business/Interfaces/IGetDepartmentByIdCommand.cs

```diff
@@ -0,0 +1,20 @@
+﻿using LT.DigitalOffice.CompanyService.Models.Dto.Models;
+using System;
+
+namespace LT.DigitalOffice.CompanyService.Business.Interfaces
+{
+    /// <summary>
+    /// Represents the command pattern.
+    /// Provides method for getting department model by id.
+    /// </summary>
+    public interface IGetDepartmentByIdCommand
+    {
+        /// <summary>
+        /// Returns the department model with the specified id.
+        /// </summary>
+        /// <param name="id">Specified id of department.</param>
+        /// <returns></returns>
+        /// /// <exception cref="Kernel.Exceptions.NotFoundException">Thrown when departmen is not found.</exception>
+        Department Execute(Guid id);
```

Comment:
Wouldn't it be better if you called the variable "departmentId"?

Reply:
no need, because of class name

Reply:
Idk, I remember Spartak was talking about excessive naming. Anyway, this interface named "IGetDepartmentByIdCommand", but honestly - I don't mind.

Reply:
Oh, ok, I named it like that in **GetDepartmentByIdCommand.cs** anyway, sorry
done